modules/op-release-notes-1-8.adoc

```diff
@@ -44,8 +44,29 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
 // (link:https://issues.redhat.com/browse/SRVKP-1782[#SRVKP-1782])
 // Writer, also see https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-nodejs/s2i-nodejs-task.yaml. Overkill: The user can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+[id="fixed-issues-1-8_{context}"]
+== Fixed issues
 
-// Pipelines
-// *
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even if GitHub Apps is not feasible.
```

Comment:
this is currently in `Fixed issues` but it is a new feature

I have added a comment describing it the doc issue 
https://issues.redhat.com/browse/RHDEVDOCS-4180
sorry missed your earlier comment.

Reply:
Updated. Thanks!